### PR TITLE
refactor: Remove node-zopfli loose dependency

### PR DESCRIPTION
**Pull request by ivasilov-h4**
As discussed in https://github.com/webpack-contrib/compression-webpack-plugin/issues/56, I've removed the node-zopfli dependency.

### CODE REVIEW

**Comment by d3viant0ne** [comment]
Note to @webpack-contrib/org-maintainers, needs to be done in conjunction with but after the first stable release of https://github.com/webpack-contrib/zopfli-webpack-plugin

**Comment by ivasilov-h4** [comment]
Zopfli-webpack plugin has been published. Is this ready to go?

File: index.js [hunk 1]
```diff
@@ -15,22 +15,7 @@ function CompressionPlugin(options) {
...
-				verbose_more: options.hasOwnProperty('verbose_more') ? options.verbose_more : false,
-				numiterations: options.numiterations ? options.numiterations : 15,
-				blocksplitting: options.hasOwnProperty('blocksplitting') ? options.blocksplitting : true,
-				blocksplittinglast: options.hasOwnProperty('blocksplittinglast') ? options.blocksplittinglast : false,
-				blocksplittingmax: options.blocksplittingmax ? options.blocksplittingmax : 15
-			};
-			this.algorithm = function (content, options, fn) {
-				zopfli.gzip(content, options, fn);
-			};
+			throw new Error("This plugin doesn't support zopfli anymore.");
```
Comment (michael-ciniawsky) [inline index.js]: Please add a note and 'link' to `zopfli-webpack-plugin` in the `err.message`

Comment (d3viant0ne) [inline index.js]: I haven't published the initial version yet, the org-maintainers need to review the new plugin before I publish an initial version. This should happen this weekend, probably Sunday.

**Comment by ivasilov-h4** [comment]
I've added a link to the zopfli plugin in the error message. This PR is good to go.

**Comment by d3viant0ne** [comment]
@ivasilov-h4 - For reference, in both @webpack & @webpack-contrib and in all honestly nearly every open source project using semver....

Don't increment a version in a pull request, not modify the changelog. That's something for the maintainers to take care of as a part of the release process.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,15 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+
+<a name="0.5.0"></a>
```
Comment (d3viant0ne) [inline CHANGELOG.md]: Revert this change

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "compression-webpack-plugin",
-  "version": "0.4.0",
+  "version": "0.5.0",
```
Comment (d3viant0ne) [inline package.json]: Revert this as well
